src/styles/main.scss

```diff
@@ -0,0 +1,157 @@
+@import 'colors';
+@import 'measures';
+
+.container {
+  padding: $paddingV-card $paddingH-card;
+  display: grid;
+  grid-template-columns: repeat(4, $width-card);
+  gap: $row-gap $column-gap;
+  justify-content: center;
+  box-sizing: border-box;
+
+  @media (max-width: 1023px) {
+    grid-template-columns: repeat(3, $width-card);
+  }
+
+  @media (max-width: 768px) {
+    grid-template-columns: repeat(2, $width-card);
+  }
+
+  @media (max-width: 488px) {
+    grid-template-columns: repeat(1, $width-card);
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  width: $width-card;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px solid $color-border-card;
+  transition: transform 0.3s ease;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    background-size: contain;
+    background-repeat: no-repeat;
+    margin: 32px 19px 36px;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    width: 166px;
+    height: 36px;
+    margin: 0 16px 4px;
+    font-family: $roboto-medium;
+    font-weight: $roboto-weight-medium;
+    color: $color-font;
+
+    &:hover {
+      color: $color-title-hover;
+    }
+  }
+
+  &__subtitle {
+    font-size: 10px;
+    line-height: 14px;
+    margin: 0 16px 16px;
+    font-family: $roboto-medium;
+    font-weight: $roboto-weight-regular;
+    min-height: 14px;
+    max-width: 98px;
+    color: $color-font-secondary;
+  }
+
+  &__reviews {
+    display: flex;
+    margin: 0 auto 24px;
+    box-sizing: border-box;
+    width: 166px;
+    height: 16px;
+    text-align: center;
+    justify-content: space-between;
+    padding: 2px 0 0 2px;
+
+    &-text {
+      width: 53px;
+      height: 16px;
+      font-family: $roboto-medium;
+      font-weight: $roboto-weight-medium;
+      font-size: 10px;
+      line-height: 14px;
+      text-align: right;
+      color: $color-font;
+    }
+
+    &-stars {
+      display: flex;
+      box-sizing: border-box;
+      width: 96px;
+      height: 16px;
+      border-radius: 0.5px;
+    }
+
+    &-star {
+      width: 16px;
+      height: 16px;
+      background-image: url(../images/star.svg);
+      background-repeat: no-repeat;
+    }
+
+    &-star:nth-child(-n + 4) {
+      margin-right: 4px;
+      background-image: url(../images/star-active.svg);
+    }
+  }
+
+  &__prices {
+    display: flex;
+    justify-content: space-between;
+    margin: 0 auto 16px;
+    align-items: center;
+    width: 166px;
+    height: 18px;
+
+    &-text {
+      font-family: $roboto-medium;
+      font-weight: 400;
```

Comment:
The font-weight for `.card__prices-text` is set to `400` which is typically the default font-weight. If this is intentional, make sure it's necessary to declare it explicitly. Otherwise, you can omit it to reduce redundancy.